content/applications/general/voip/axivox/dial_plan_advanced.rst

```diff
@@ -0,0 +1,289 @@
+24 hours a day, 7 days a week. Using *Axivox* advanced dial plan features, the process can be
+automated and routing can be setup for all scenarios. This way customers are never left waiting or
+frustrated because they can not get in touch with anyone.
+
+Using the advanced elements in dial plans, companies can automate call routing for certain days or
+times, like company holidays. Companies can also allow callers to enter extensions themselves and
+get transferred automatically using a digital receptionist, so an administrative team does not have
+to be available around the clock. There is even the option to route callers depending on where they
+are calling from in the world, to maximize efficiency.
+
+.. seealso::
+   For more information on basic dial plans and adding elements visit :doc:`dial_plan_basics`.
+
+Advanced elements
+=================
+
+In Axivox dial plans there are two advanced elements that can be used.
+
+- :guilabel:`Record`: recording feature is enabled (requires plan change, enabled in Settings).
+- :guilabel:`Caller ID`: replace the caller ID by the called number or free text.
+
+To add one of these elements navigate to :guilabel:`Dial plans` in the left menu of the `Axivox
+management console <https://manage.axivox.com>`_. The :guilabel:`Record` element records calls that
+are routed through this element and requires an additional plan change in Axivox.
+
+To enable recording on Axivox, navigate to :guilabel:`Settings` in the `Axivox management console
+<https://manage.axivox.com>`_. Then go to the :guilabel:`Recording` drop-down menu near the bottom
+of the page. Select :guilabel:`Enabled` to enable recording using the :guilabel:`Record` element in
+a dial plan.
+
+The :guilabel:`Caller ID` element allows for the replacement of the caller ID downstream, after
+routing. Upon adding the :guilabel:`Caller ID` element and double clicking it to configure it, two
+options appear. The first is a :guilabel:`Free text` field where any text can be input to replace
+the caller ID. The second option is :guilabel:`Replace the caller ID by the called number`. This
+option replaces the caller's ID with the :guilabel:`Incoming number`.
+
+.. important::
+   A company may want to use the :guilabel:`Caller ID` element to replace the :guilabel:`Incoming
+   number` so that employees or outside transfers cannot see the number and information is kept
+   private.
+
+Basic routing elements
+======================
+
+Basic routing elements in Axivox dial plans provide extension based routing. This can be done by
+adding either a :guilabel:`Menu` to numerically link the dial-by-numbers to an action, or by using
+a :guilabel:`Digital Receptionist` to automatically route or listen for an extension based on a key
+input from the caller. The main difference between the two elements is that the :guilabel:`Digital
+Receptionist` does **not** need to be pre-configured numerically with actions. Instead, it acts as a
+virtual receptionist.
```

Comment:
```suggestion
Basic routing elements in Axivox dial plans provide extension-based routing. This can be done by
adding either a *Menu* to numerically link the dial-by-numbers to an action, or by using a *Digital
Receptionist* to automatically route, or listen for an extension based on a key
input from the caller.

The main difference between the two elements is that the Digital Receptionist does **not** need to
be pre-configured numerically with actions. Instead, it acts as a virtual receptionist.
```

Reply:
Where can any of these features/elements be accessed? I'm not seeing this information/option(s)...

Reply:
I've added a basic process into each section

Reply:
Good call on explaining the menu feature a bit better